readme.md

```diff
@@ -98,6 +98,14 @@ On top of that `lightweight` integration with build tools is provided in order t
 Lightweight integration with build tools, refers to reading information from the build tool config without bringing in the build tool itself into the classpath.
```

Comment:
Is it possible to add a concrete example containing `pom.xml` snippet code and what the generated resource will then include coming from the pom.xml's file ?

Reply:
TBH I feel that saying that we just read `artifactId` and `version` that we use as lables and to name resources suffice like images, deployments, service etc suffice.